content/doc/book/security/securing-org-folders-and-multibranch-pipelines.adoc

```diff
@@ -0,0 +1,84 @@
+---
+title: Securing SCM credentials for Organization Folders and Multibranch Pipelines
+layout: section
+---
+ifdef::backend-html5[]
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+Organization Folders and Multibranch Pipelines are Jenkins project types that allow a single configuration item to automatically manage and build a potentially large number of Pipeline jobs for different repositories or different branches of a single repository.
+These projects interact with external SCM providers in various ways, which generally require credentials with SCM provider API access to be made available to Jenkins ("scan credentials").
+The following sections discuss potential issues with visibility and accessibility of those credentials.
+
+== Credentials
+
+For many Branch Source configurations, such as those provided by GitHub Branch Source plugin and Bitbucket Branch Source plugin, the Jenkins user who configures the project only needs to provide a single credential, which will be used in two ways:
+
+* By the controller, to perform plugin operations like Organization Folder scans, Multibranch Pipeline branch indexing, webhook modification, commit status updates, and more. These operations generally interact with SCM provider APIs that may require significant permissions beyond read/write access to the contents of a repository.
+* By nodes, to check out the repositories being built by the child Pipeline jobs.
+
+Credentials defined using Global scope are available to all children of the object where they are defined.
+This means that any credentials available to Organization Folders and Multibranch Pipelines will also be available to their child Pipeline jobs.
+
+Any user who has Job/Create permission in a location where a credential is available must be trusted to use that credential arbitrarily (and in many cases, Job/Configure is also enough to use credentials arbitrarily).
```

Comment:
```suggestion
Any user who has Job/Create permission in a location where a credential is available must be trusted to use that credential arbitrarily (and in many cases, Job/Configure is enough to use credentials arbitrarily).
```
Suggestion to remove "also" unless it should absolutely describe that this would only be in addition to something else.